docs/DeveloperGuide.adoc

```diff
@@ -127,8 +127,7 @@ The `Model`,
 * does not depend on any of the other three components.
 
 [NOTE]
-As a more OOP model, we can store a `Tag` list in `Address Book`, which `Order` can reference. This would allow `Address Book` to only require one `Tag` object per unique `Tag`, instead of each `Person` needing their own `Tag` object. An example of how such a model may look like is given below. +
- +
+Better Model Class Diagram
 image:BetterModelClassDiagram.png[]
```

Comment:
Please update the BetterModelClassDiagram.png after you updated the uml diagram.